Active Documents/contenteditable-disabled.html

```diff
@@ -0,0 +1,192 @@
+        </p>
+        <figure>
+            <img src="images/mac-touchbar.png" alt="Mac Touch Bar showing editing buttons">
+            <figcaption>
+                Mac Touch Bar showing editing buttons
+            </figcaption>
+        </figure>
+        <figure>
+            <img src="images/ios-editing-buttons.png" width="300" alt="iOS selection menu showing editing buttons">
+            <figcaption>
+                iOS selection menu showing editing buttons
+            </figcaption>
+        </figure>
+        <p>
+            To address the issue, a new contenteditable-disabled attribute can be applied as shown in the example below.
+        </p>
+        <pre class="example">
+            &lt;div contenteditable="true" contenteditable-disabled="
+                formatbackcolor
+                formatdent
+                formatfont
+                formatjustify
+                formatscript
+                formatstrikethrough
+                formatunderline
+                inserthorizontalrule">
+                This is a contenteditable region of a document that supports:
+                &lt;ul>
+                    &lt;li>bold text&lt;/li>
+                    &lt;li>italic text&lt;/li>
+                    &lt;li>links&lt;/li>
+                    &lt;li>lists&lt;/li>
+                    &lt;li>and possibly other formatting outside the scope of this specification.&lt;/li>
+                &lt;/ul>
+                
+                This contenteditable region explicitly does not support:
+                &lt;ul>
+                    &lt;li>background colors&lt;/li>
+                    &lt;li>indentation&lt;/li>
+                    &lt;li>font name, size or color customization&lt;/li>
+                    &lt;li>justification&lt;/li>
+                    &lt;li>superscript or subscript text&lt;/li>
+                    &lt;li>strikethrough&lt;/li>
+                    &lt;li>underlines&lt;/li>
+                    &lt;li>horizontal rules&lt;/li>
+                &lt;/ul>
+            &lt;/div>
+        </pre>
+    </section>
+    <section id="contenteditable-disabled">
+        <h2>
+            <code>contenteditable-disabled</code> DOM Content Attribute.
+        </h2>
+        <p>
+            The <dfn><code>contenteditable-disabled</code></dfn> attribute indicates which user agent-supplied editing UI SHOULD be disabled.  If specified, the attribute must have a value that is an <a href=https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#unordered-set-of-unique-space-separated-tokens>unordered
+            set of unique space-separated tokens</a> [[HTML]] which
+            are <a href=https://infra.spec.whatwg.org/#ascii-case-insensitive>ASCII case-insensitive</a> [[INFRA]].
```

Comment:
```suggestion
            The <code><dfn data-dfn-type="element-attr">contenteditable-disabled</dfn></code> attribute indicates which user agent-supplied editing UI SHOULD be disabled.  When specified, the attribute can contain an [=unordered set of unique space-separated tokens=] that are treated as [=ASCII case-insensitive=].
```

Reply:
Note: "unordered set of unique space-separated tokens" hans't been exported yet from HTML... still need to do that. In any case, please avoid liking directly to the HTML spec using `<a href=>`, always use `[=whatever you want to link to=]`... and if `[=whatever you want to link to=]` doesn't exist in the [xref database](http://respec.org/xref/), we need to raise an issue on the corresponding spec to get the term exported and added to the database.

Reply:
yes, ReSpec now throws the error. I will create an issue once the first change is landed so I can reference updated draft in the issue. [Issue](https://github.com/w3c/editing/issues/258) tracking the work here.